.travis.yml

```diff
@@ -105,7 +105,21 @@ install:
   - rm configure
   - R CMD INSTALL .
   - popd
-
+  - pushd $HOME
+  # Install preles package
+  - git clone https://github.com/MikkoPeltoniemi/Rpreles
+  - cd Rpreles
+  - R CMD INSTALL .
+  - popd
+  - pushd $HOME
+  # Install linakages package
+  - git clone https://github.com/araiho/linkages_package
+  - cd linkages_package
+  - R CMD INSTALL .
+  - popd
+  #install remotes if version less than or equal 2.0.1
```

Comment:
More than the version number (most folks can probably infer that from the code below), would probably be good to record the reason. Maybe `# temporary until remotes 2.0.2 binary reaches C2D4U`?